---
timestamp: '2025-01-16T23:35:30.289323'
level: INFO
issuer: instance_solver
event_type: metadata
data:
    log_file: /home/remote/u7899572/lifted-white-plan-domain-repair/exp_logs_anu/astar-zeroary
        relax-prec-delete lp1/mystery__pprob19-err-rate-0-1.yaml
    instance_id: mystery__pprob19-err-rate-0-1
    search_algorithm: astar
    plan_length: 6
    domain_class: mystery
    instance_name: pprob19-err-rate-0-1
    ground_plan:
    - (feast lubricity lemon beef bavaria manitoba)
    - (overcome abrasion lubricity beef saturn earth)
    - (feast lubricity beef lamb pennsylvania oregon)
    - (feast lubricity lamb haroset goias bosnia)
    - (feast lubricity haroset lobster bavaria manitoba)
    - (succumb abrasion lubricity lobster saturn earth)
    lifted_plan:
    -   - feast
        - ?lubricity
        - ?lemon
        - ?beef
        - ?bavaria
        - ?manitoba
    -   - overcome
        - ?abrasion
        - ?lubricity
        - ?beef
        - ?saturn
        - ?earth
    -   - feast
        - ?lubricity
        - ?beef
        - ?lamb
        - ?pennsylvania
        - ?oregon
    -   - feast
        - ?lubricity
        - ?lamb
        - ?haroset
        - ?goias
        - ?bosnia
    -   - feast
        - ?lubricity
        - ?haroset
        - ?lobster
        - ?bavaria
        - ?manitoba
    -   - succumb
        - ?abrasion
        - ?lubricity
        - ?lobster
        - ?saturn
        - ?earth


---
timestamp: '2025-01-16T23:35:31.236984'
level: INFO
issuer: instance_solver
event_type: ground_repair
data:
    repair_length: 1
    repair: feast,(pleasure ?l1),precPos,-1


---
timestamp: '2025-01-16T23:35:34.199824'
level: ERROR
issuer: instance_solver
event_type: error
data: "An error occurred during A* search: 'Node' object has no attribute 'neighbours'.\
    \ Stack trace: Traceback (most recent call last):\n  File \"instance_solver.py\"\
    , line 104, in solve_instance\n    searcher.find_path(logger=logger, log_interval=log_interval)\n\
    \  File \"/home/remote/u7899572/lifted-white-plan-domain-repair/search_partial_grounding/search_algorithms.py\"\
    , line 95, in find_path\n    self.log_iteration_info(logger, iteration, open_list,\
    \ current_node, final=True, is_goal=False)\n  File \"/home/remote/u7899572/lifted-white-plan-domain-repair/search_partial_grounding/search_algorithms.py\"\
    , line 37, in log_iteration_info\n    \"current_node\": current_node.to_dict(include_state=False)\n\
    \  File \"/home/remote/u7899572/lifted-white-plan-domain-repair/search_partial_grounding/node.py\"\
    , line 233, in to_dict\n    \"num_neighbours\": len(self.neighbours),\nAttributeError:\
    \ 'Node' object has no attribute 'neighbours'\n"


---
timestamp: '2025-01-16T23:35:34.353278'
level: INFO
issuer: instance_solver
event_type: timer_seconds
data: '116.92'


---
timestamp: '2025-01-16T23:37:11.253410'
level: INFO
issuer: batch_solver
event_type: general
data: 'Subprocess finished successfully. Duration: 253.08 seconds'


